### ISSUE TITLE
Use font with wide scripts support

### ISSUE THREAD

**Issue by sergeevabc** [go-shiori/shiori#537]
Shiori 1.5.3 on Windows 7 x64.

At present the app uses custom font with Latin script support only.
To be specific: ```/css/fonts/source-sans-pro-v13-latin-700.woff2```
And what do others see when they use letters out of Latin scope? The eyesore.

![image](https://user-images.githubusercontent.com/373718/201513100-1cdbd151-a731-477e-8f20-86f6592dad97.png)

Either use font with wide scripts support (e.g. Noto Sans) or let system/user decide.

**Comment by fmartingr** [comment]
Hey @sergeevabc, I'm completely unfamiliar with this, I have some japanese bookmarks but I have not seen this issue on those. Are there built-in fonts on operating system that support wide-script support? So we could fallback to those or stop using custom fonts altogether.

**Comment by jansol** [comment]
Simply using the `serif`/`sans-serif`/`monospace` aliases should yield a decent set of fonts and their corresponding fallback lists as determined by the browser/system.

**Comment by fmartingr** [comment]
> Simply using the `serif`/`sans-serif`/`monospace` aliases should yield a decent set of fonts and their corresponding fallback lists as determined by the browser/system.

By that you mean that we can use:

```less
font-family: Arial, serif, sans-serif; // or monospace where applicable 
```

(Where Arial can be any ohter font)

And that would work? I see we already have `sans-serif` set as fallback in most places:

```
$ ag "font-family" --file-search-regex '\.(less|html)$'
internal/view/assets/less/archive.less
46:                     font-family: "Source Sans Pro", sans-serif;

internal/view/assets/less/common.less
62:     font-family: "Source Sans Pro", sans-serif;
476:                    font-family: "Ubuntu Mono", "Courier New", Courier, monospace;

e2e/playwright/test-results/report.html
7:        body { font-family: Arial, sans-serif; margin: 20px; }

e2e/playwright/test-results/report copy.html
7:        body { font-family: Arial, sans-serif; margin: 20px; }

e2e-report.html
7:        body { font-family: Arial, sans-serif; margin: 20px; }
```

**Comment by jansol** [comment]
> By that you mean that we can use:

Yes, although it shouldn't be necessary to specify both `serif` and `sans-serif` (and `monospace` for that matter), any one of the three should be enough as a fallback.
